### ISSUE TITLE
Tags displayed in the popup window

### ISSUE THREAD

**Issue by xueyunquan** [adamvoss/vscode-languagetool#10]
In VS Code Insider 1.17.10 x64 for Windows, in a plain text file, when there is some certain grammar error in the file, the popup window shows some tags, that I believe shouldn't show up. Please refer to the screenshot below:

![tmp20170918](https://user-images.githubusercontent.com/4216222/30527798-fc9a10ec-9c5e-11e7-8c8e-865de39058ca.png)

Here I think the `<suggestion></suggestion>` tags should not be shown. Please look into this issue.

Thank you.

**Comment by adamvoss** [comment]
I agree doing some processing of xml tags when present, rather than rendering them directly would be a better user experience.  Do you have a suggestion on how they be rendered?
